test/unit/configuration-test.js

```diff
@@ -181,8 +200,11 @@ describe('configuration._coerceRemovedOptions()', () => {
     it('gets coerced to color set to boolean true', () => {
       assert.deepEqual(config, { options: { color: true } });
     });
+    it('produces no warnings', () => {
+      assert.lengthOf(coerceResult.warnings, 0);
+    });
     it('produces no errors', () => {
-      assert.deepEqual(coerceResult.errors, []);
+      assert.lengthOf(coerceResult.errors, 0);
```

Comment:
I used `deepEqual` as in this case it provides richer diff when the test doesn't pass (it tells you exactly what's wrong instead of just saying 1 != 0), but since everyone is correcting this and pointing it out in reviews, I'm giving up on it 😄 I guess it's not immediately obvious for anyone else, so probably not worth keeping.

Reply:
Oh, I see. Well, I had no hard opinion on this. The only reason I changed it is because errors/warnings assertions were different through test cases, while testing seemingly same output. I'm okay with using `deepEqual` for *both* of them, if that makes a better DX during test failure debugging.

Reply:
I'm fine with leaving lengthOf there. You're not the first one confused by the mixed usage of lengthOf and deepEqual.